src/layout/section/seaSection/SeaSection.style.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components'
+
+import { color, font, responsiveBreakpoints, space } from '../../../_utils/branding'
+import { TextSubHeader } from '../../../typography/subHeader'
+
+const CONTENT_WIDTH = '375px' // iPhone6 width same as componentSizes.searchOverlayWidth
```

Comment:
I'm not a fan of sharing a variable just because they have the same value. Are we sure it was intended on the design side? If we update the `searchOverlayWidth` for a specific reason de we want it to update this screen as well? I would keep it local `CONTENT_WIDTH` variable here.

Reply:
Alright, I've checked where `searchOverlayWidth` is used and yes it is definitely not link to a device size, so I agree, let's keep `CONTENT_WIDTH`

Reply:
I think many of the entries in `componentSizes` object shouldn't be there as they are not global

Reply:
Can you revert based on what we said?

Reply:
Ok! I'll revert and let the comment to keep a track on where we are using same values (looking for patterns)